scripts/config.tests.json

```diff
@@ -57,7 +57,8 @@
         "relay_resolver_enable_interface_output_type": { "kind": "enabled" }
       },
       "language": "flow",
-      "experimentalEmitSemanticNullabilityTypes": true
+      "experimentalEmitSemanticNullabilityTypes": true,
+      "liveResolverContextType": { "name": "LiveResolverContextType", "path": "packages/relay-runtime/mutations/__tests__/LiveResolverContextType" }
```

Comment:
Do we need another project that exercises the `package` module specifier variant?

Reply:
Possibly? As far as I could find there was only one project config for all tests